src/Counter/ButtonList.js

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+
+function ButtonList(onClickButton) {
+    return <div>
+        <button className="btn btn-primary" name="inc" onClick={() => onClickButton.onClickHandler(1)}>Increment</button>
+        <button name="dec" className="btn btn-primary" onClick={() => onClickButton.onClickHandler(-1)}>Decrement</button>
```

Comment:
let's create handlers for onClick logic 

e.g. 
```
const handleButtonClick = () => {
  onClickButton.onClickHandler(-1);
}
```

onClick={handleButtonClick}

Reply:
also, we don't need this separate component for now, let's put all connected to the task about Counter in one component

Reply:
fixed